angular/src/app/modules/review/review-detail/review-detail.component.css

```diff
@@ -200,6 +200,22 @@ body{
     background-color: #2196f3!important;
 }
 
+.btn-pm-head{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 160px;
+    margin: 8px auto 0px;
+    height: 35px;
+    text-align: right;
+    padding-right: 20px;
+    color : #ffff;
+    background-color: #0b984a!important;
+}
```

Comment:
Should move the common button style to a common class and then inherited from that class
```
display: flex;
    justify-content: space-evenly;
    align-items: center;
    margin: 8px auto 0px;
    height: 35px;
    text-align: right;
    padding-right: 20px;
```